presto-main/src/test/java/io/prestosql/operator/scalar/TestMathFunctions.java

```diff
@@ -795,6 +795,40 @@ public void testRound()
         assertFunction("round(REAL '-3.5001', 1)", REAL, -3.5f);
         assertFunction("round(REAL '-3.99', 1)", REAL, -4.0f);
 
+        // ROUND negative DECIMAL
+        assertFunction("round(TINYINT '9', -1)", TINYINT, (byte) 10);
+        assertFunction("round(TINYINT '-9', -1)", TINYINT, (byte) -10);
+        assertFunction("round(TINYINT '-14', -1)", TINYINT, (byte) -10);
+        assertFunction("round(TINYINT '12', -1)", TINYINT, (byte) 10);
+        assertFunction("round(TINYINT '18', -1)", TINYINT, (byte) 20);
+        assertFunction("round(TINYINT '18', -2)", TINYINT, (byte) 0);
+        assertFunction("round(TINYINT '18', -3)", TINYINT, (byte) 0);
```

Comment:
Add a test cases:
- `round(TINYINT '127', -3)` -> overflow
- `round(TINYINT '-128', -3)` -> overflow

Reply:
> round(TINYINT '127', -3) -> overflow

Are you sure this should overflow? Rounding to -3 will return a value of 0. Rounding to -1 will return 130 and should overflow.

Reply:
I made same mistake again. You're correct.